tests/test_macros.py

```diff
@@ -73,26 +73,19 @@ def test_pdf_extension(testapp, webassets, record):
         assert "pdf-file-uri" in res.get_data(as_text=True)
 
 
-def test_csv_dthreejs_extension(testapp, webassets, record):
+def test_csv_papaparsejs_extension(testapp, webassets, record):
     """Test view with csv files."""
     create_file(record, "test.csv", BytesIO(b"A,B\n1,2"))
     with testapp.test_client() as client:
         res = client.get(preview_url(record["control_number"], "test.csv"))
         assert 'data-csv-source="' in res.get_data(as_text=True)
-        assert 'data-csv-delimiter=","' in res.get_data(as_text=True)
 
-        with patch("csv.Sniffer", side_effect=Exception):
```

Comment:
Please, note this change in the test: The new library does not parse the csv on the server side. Everything is done on the client, and with streaming. So, the `delimiter` is not passed by the server any more, and the error in decoding is not passed by the server